docs/COMPATIBILITY.md

```diff
@@ -109,14 +109,14 @@ Note that HHVM is only required if you do not use PHP.
 	</tr>
 	<tr>
 		<th>SMW 3.1.x</th>
-		<td><strong><a href="https://php.net/supported-versions.php">7.1.0</a></strong> - latest</td>
-		<td>No support</td>
-		<td><strong><a href="https://www.mediawiki.org/wiki/Version_lifecycle">1.31.0</a></strong> - ?</td>
+		<td><strong><a href="https://php.net/supported-versions.php">7.0.0</a></strong> - latest</td>
+		<td>No support<sup>[t.0]</sup></td>
+		<td><strong><a href="https://www.mediawiki.org/wiki/Version_lifecycle">1.31.0</a></strong> - 1.33.x</td>
 	</tr>
 	<tr>
 		<th>SMW 3.0.x</th>
-		<td><strong><a href="https://php.net/supported-versions.php">5.6.0</a></strong> - latest</td>
-		<td>3.5.0 - latest</td>
+		<td><strong><a href="https://php.net/supported-versions.php">5.6.0</a></strong> - 7.2.x</td>
+		<td>3.5.0 - 3.30.x</td>
```

Comment:
Updated the wrong line

Reply:
Oh this is about HHVM... nvm then

Reply:
Yeah, and 3.30 is a hard ending since this is the last version supporting php

Reply:
To be honest I'd just drop the HHVM stuff and let people that want to use it (maybe no one) figure out compat based on the PHP versions. That is what I did in https://github.com/JeroenDeDauw/Maps/blob/master/INSTALL.md#platform-compatibility-and-release-status

Reply:
Fair enough, I will use the Maps' INSTALL.md as an inspiration to redo the table and merge them into one excluding HHVM

Reply:
So I reworked the whole document. I is a bit more radical that perhaps expected but I believe the document gained in clarity. I kicked everything below the 1.9.0 release to make the whole thing more sane. Ancient info is still available on wiki.

Reply:
I think you add a class attribute to the table so that when the content is included into a wiki page it use that class instead of the vanilla "i-am-not-a-table-and-i-look-like-...".

Reply:
Yeah, I guess you saw me fiddle with the compat page the other day. :)